_posts/2019-09-30-team-working-agreements.md

```diff
@@ -0,0 +1,261 @@
+obvious to put in a team working agreement, but it portrays clearly to everyone that they should attend all sprint
+meetings and be actively engaged.
+
+The same team got a lot out of making slogans for the items in their working agreement. Phrases like "Pair by
+default" and "Incremental improvement over consistency" were introduced during the brainstorm. They've since become
+mantras for the team to refer to during sprint work and ceremonies. Team values and habits solidify through this
+kind of repetition.
+
+### The team working agreement provides on-boarding documentation.
+
+When new members join a team, it takes time for them to adjust. A team working agreement provides an up-to-date
+reference on the team's preferred methods of working. The working agreement isn't set in stone, and as the team
+changes it's important to update it...but having it in writing helps new members acclimate quickly.
+
+## What are some challenges with team working agreements?
+
+### Uncertainty about what the team agreement should include.
+
+Some teams have struggled to identify what belongs on a team working agreement. Should it describe our team values?
+Habits? Procedures? Rules?
+
+It likely varies from team to team what you want here. (ADAM: provide some examples of how it may vary for some
+type of teams.) It's definitely important that you identify what you're looking for up front, and make it clear
+heading into the brainstorm. Suggest categories, sample habits, & questions to ask heading into the brainstorm, and
+keep them visible during the meeting.
+
+Be cautious about introducing individual bias, though.
+
+<aside class="dialogue">
+  <div class="answer">
+    <img src="/images/dialogue/steve.jpg">
+    <p class="intro">Steve Hicks says...</p>
+    <p>
+      During my first team agreement brainstorm, I felt like we spent a lot of time talking about details 
+      related to specific incidents. On my next team, I made a strong effort to constrain the suggestions 
+      to avoid this. To me, team values were really important for this new team - and I did such a good 
+      job of encouraging those types of suggestions that it's pretty much all we ended up with.
+    </p>
+    <p>
+      While this made me feel like I understood the values of my team better, we're months later _still_ 
+      paying for this bias. Our working agreement lacked important procedural and tactical definition. We've 
+      been adding things like JIRA hygiene and responsibilities of project leaders as they come up, but I would
+      not push a working agreement brainstorm so strongly in one direction again.
+    </p>
+  </div>
+</aside>
+
+(ADAM: Add an Adam says... section to share the habits that the Galleries team agreed upon)
+
+Remember that the working agreement is a living document. If the team learns that it missed something in the
+brainstorm, don't hesitate to update the agreement.
```

Comment:
Where/how do updates occur? This is a good spot to suggest revisiting working agreements in retros.

Reply:
Good point - I will add a sentence recommending retros as a good time.

Reply:
```suggestion
brainstorm, don't hesitate to update the agreement. Team retros are a great time to make updates.
```